crates/ruff_python_formatter/src/options.rs

```diff
@@ -246,3 +260,18 @@ impl FromStr for MagicTrailingComma {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Default)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum PreviewMode {
+    #[default]
+    Disabled,
+
+    Enabled,
+}
+
+impl PreviewMode {
+    pub const fn is_enabled(self) -> bool {
+        matches!(self, PreviewMode::Enabled)
+    }
+}
```

Comment:
Why this instead of deriving the `is` macro?

Reply:
Because I and the `is` macro aren't good friends ;)

Reply:
Hm could you change the other one for consistency then?

Also why lol

Reply:
Subscribe, I like the `is` macro :joy:

Reply:
damn, I'm outpowered